components/agenda.tsx

```diff
@@ -136,36 +137,34 @@ const agenda = (WrappedComponent: React.ComponentType<AgendaProps>, externalProp
                 <em>Error loading this session</em>
               </Fragment>
             )}
-            {isLoading === false &&
-              isError === false &&
-              session && (
-                <Fragment>
-                  <strong>
-                    {props.isKeynote
-                      ? 'KEYNOTE - '
-                      : props.isLocknote
-                        ? 'LOCKNOTE - '
-                        : props.isLunchnote
-                          ? 'LUNCHNOTE - '
-                          : null}
-                    {session.Presenters.map(p => p.Name).join(', ')}
-                  </strong>
-                  <br />
-                  <em>{session.Title}</em>
-                  {props.sponsorName && (
-                    <Fragment>
-                      <br />
-                      <small>Sponsored by: {props.sponsorName}</small>
-                    </Fragment>
-                  )}
-                  {props.room && (
-                    <Fragment>
-                      <br />
-                      <small className="room">{props.room}</small>
-                    </Fragment>
-                  )}
-                </Fragment>
-              )}
+            {isLoading === false && isError === false && session && (
```

Comment:
Still wondering if the prettier settings are ok or prettier just wasn't previously used despite the settings being there